components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -418,6 +433,14 @@ private static void setReCaptchaConfigs(Properties properties) {
         }
         CaptchaDataHolder.getInstance().setReCaptchaRequestWrapUrls(reCaptchaRequestWrapUrls);
 
+        String reCaptchaScoreThreshold = properties.getProperty(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD);
+        try {
+            CaptchaDataHolder.getInstance().setReCaptchaScoreThreshold(Double.parseDouble(reCaptchaScoreThreshold));
+        } catch (NumberFormatException | NullPointerException e) {
```

Comment:
Don't have NPE error. those should be captured by checking whether they are null or empty